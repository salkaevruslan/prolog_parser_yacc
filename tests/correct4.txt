f :- a ((((b)))).
f :- a , b.