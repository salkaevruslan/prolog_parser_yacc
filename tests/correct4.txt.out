relation
	head
		atom
			identifier = f
	body
		atom
			identifier = a
			atom
				identifier = b
relation
	head
		atom
			identifier = f
	body
		conjunction
			atom
				identifier = a
			atom
				identifier = b
